EnglishFilter/sections/foreign.txt

```diff
@@ -9261,6 +9261,8 @@ brandbuffet.in.th#@#.category-advertorial
 !------------------------------ Vietnam ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/105246
+voz.vn##.adsbypubpower
```

Comment:
This rule already exists in ABPVN List, there is no need to add it to our filter.